LICENSE

```diff
@@ -0,0 +1,35 @@
+Copyright (C) 2013-2016 Open Source Robotics Foundation, Inc; CA, USA
+Copyright (C) 2016 Janko Petereit <janko.petereit@gmail.com>
```

Comment:
I added this based on "git log|grep ^Author:|sort -u" ;-) It was just meant to be an example file, I never intended this to be included as-is.

That said, this kind of issue highlights one of the problems with doing this after the fact: If you have to look at the commit itself to decide whether it implies copyright or not, it doesn't scale well. For catkin, I already weeded out one such case when looking at it, but for gencpp I missed it. Also, it becomes a judgment call quickly: What do you do if one person adds 100 one-liners? Does that still not qualify?